### PR TITLE
Extract list of PoPs into a constant + Add Paris, FR

### PR DESCRIPTION
**Pull request by san983**
## 👓 Preview

Anycast_DNS_-_DNSimple_Help

### CODE REVIEW

**Review by aeden — APPROVED** [review]

File: content/articles/anycast.markdown [hunk 1]
```diff
@@ -9,13 +9,10 @@ categories:
...
 
-- San Jose, CA, USA
-- Chicago, IL, USA
-- Ashburn, VA, USA
-- Amsterdam, Netherlands
-- Tokyo, Japan
-- Sydney, Australia
+<% POPS.each do |server| %>
+* <%= server %>
+<% end %>
```
Comment (dallasread) [inline content/articles/anycast.markdown:16]:
```suggestion
<% POPS.each do |server| %>* <%= server %>
<% end %>
```

**Review by dallasread — APPROVED** [review]
⚡  Looks good to me!

I made a small suggestion in case you want to get rid of the inner `<p>` tag to remove the extra spacing (there are probably better ways to do it though?).

File: content/articles/regional-records.markdown [hunk 2]
```diff
@@ -13,14 +13,11 @@ This article describes a feature only available on the following plans: Professi
...
-* Illinois, US
-* Virginia, US
-* Amsterdam, NL
-* Tokyo, JP
-* Sydney, AU
+We have <%= POPS.count %> points of presence:
+
+<% POPS.each do |server| %>
+* <%= server %>
+<% end %>
```
Comment (dallasread) [inline content/articles/regional-records.markdown:20]:
```suggestion
<% POPS.each do |server| %>* <%= server %>
<% end %>
```

File: content/articles/where-are-your-servers-located.markdown [hunk 3]
```diff
@@ -1,18 +1,14 @@
...
-* San Jose, CA, US
-* Chicago, IL, US
-* Washington DC, US
-* Amsterdam, Netherlands
-* Frankfurt, Germany
-* Tokyo, Japan
-* Sydney, Australia
+<% POPS.each do |server| %>
+* <%= server %>
+<% end %>
```
Comment (dallasread) [inline content/articles/where-are-your-servers-located.markdown:14]:
```suggestion
<% POPS.each do |server| %>* <%= server %>
<% end %>
```

File: content/articles/where-are-your-servers-located.markdown [hunk 4]
```diff
@@ -1,18 +1,13 @@
 ---
 title: Where are your servers located?
-excerpt: Our Anycast network consists of 7 points-of-presence around the world.
+excerpt: Our Anycast network consists of <%= servers.count %> points-of-presence around the world.
```
Comment (san983) [inline content/articles/where-are-your-servers-located.markdown:3]:
```suggestion
excerpt: Our Anycast network consists of <%= POPS.count %> points-of-presence around the world.
```

**Pull request merged**